clients/osv/src/main/kotlin/OsvApiClient.kt

```diff
@@ -87,7 +87,7 @@ interface OsvApiClient {
 }
 
 @Serializable
-data class VulnerabilitiesForPackageRequest private constructor(
+class VulnerabilitiesForPackageRequest private constructor(
```

Comment:
> Use a normal `class` instead of a `data class` to not generate the
`copy()` function.

Please explain in the commit message *why* not generating the `copy()` function is a desirable thing here.